livekit-api/src/services/egress.rs

```diff
@@ -215,6 +219,8 @@ impl EgressClient {
                     segment_outputs,
                     image_outputs,
                     output: None, // Deprecated
+                    // TODO implement webhooks
+                    webhooks: Default::default(),
```

Comment:
The reason why we have to do this everytime is because we're using the wrong syntax
```suggestion
                    ..Default::default(),
```